### PR TITLE
Display uneditable habit counter reset frequency to user #8571

### LINKED ISSUE #8571: Habits from Challenges and Group Plans should show counter reset frequency (uneditable)
[//]: # (Before logging this issue, look through common problems at https://github.com/HabitRPG/habitrpg/issues If you find your issue there, read at least the first post to see if there is a workaround for you)

[//]: # (Github is primarily used for reporting bugs. If you have a feature request, use "Help > Request a Feature" so that the feature request can be vetted by the larger Habitica community)

[//]: # (To report a bug in one of the mobile apps, please report it in the correct repository. Android: https://github.com/HabitRPG/habitrpg-android, iOS: https://github.com/HabitRPG/habitrpg-ios)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitrpg/issues/2760)

[//]: # (Fill out relevant information - UUID is found in Settings -> API)

### Description
[//]: # (Describe bug in detail here. Include pictures if helpful.)

The new counters on Habits cannot be edited if the task originates from a Challenge or a Group Plan; that is expected behavior. However, it would be valuable for a user to be able to _see_ the reset frequency. Two possibilities, both of which could be implemented:

* In the tooltip that currently just says "Counter", it could say "Counter (resets Xly)" with the appropriate interval for X. (This would apply in all cases, not just Challenge/Group tasks.)
* The Advanced Options in the task edit modal could show the selected frequency, but disabled for editing. (Just for Challenge/Group tasks.)

### PR DESCRIPTION
**Pull request by yugensoft**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes https://github.com/HabitRPG/habitica/issues/8571

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)
Made uneditable habits (e.g. created by challenges) show counter reset frequency
Added counter reset frequency to counter tooltip

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID: 6365790b-b0ea-4da6-bbe3-f5a217330ad2

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
I'm not sure this disables the counters. I'll have to check locally, but I believe from the code, the user will still be able to mess with the settings on the client.

**Comment by yugensoft** [comment]
@TheHollidayInn the form control becomes visible but remains greyed-out due to the ng-disabled directive: `select.form-control(ng-model='task._edit.frequency', ng-disabled='!canEdit(task)')`

**Comment by TheHollidayInn** [comment]
Ah, perfect. I missed that.

**Comment by TheHollidayInn** [comment]
Works locally form me

**Comment by SabreCat** [comment]
Merged, thank you! This will go live after a couple weeks of testing in a staging environment.

Because this was your first accepted PR, you have earned Blacksmith tier 1! Note that further tiers require increasing amounts of effort from one to the next, so tier 2 may require multiple PRs or a larger PR to attain. But continue helping out and we will express our gratitude accordingly!

File: website/views/shared/tasks/meta_controls.jade [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
   // Counter
   span(ng-if='showDoubleTaskCounter(task, obj)')
-    span(tooltip=env.t('habitCounterUp')) +{{task.counterUp}}|
-    span(tooltip=env.t('habitCounterDown')) -{{task.counterDown}}&nbsp;
+    span(tooltip='{{env.t("habitCounterUp")}} ({{env.t("resets")}} {{env.t(task.frequency)}})') +{{task.counterUp}}|
```
Comment (khipkin) [inline website/views/shared/tasks/meta_controls.jade:5]: This is not valid localization; concatenating individually localized words/phrases can result in messages that don't read well in other languages.

Comment (SabreCat) [reply website/views/shared/tasks/meta_controls.jade:5]: Ah, good catch! Yes, this should be a single base string with variables, not three in order.

Comment (yugensoft) [reply website/views/shared/tasks/meta_controls.jade:5]: Fair call, whats the normal procedure for fixing an already merged pull request?

Comment (SabreCat) [reply website/views/shared/tasks/meta_controls.jade:5]: Go ahead and submit a fix PR and we'll roll it in before the main PR goes into production!

Comment (yugensoft) [reply website/views/shared/tasks/meta_controls.jade:5]: OK it's done https://github.com/HabitRPG/habitica/pull/8698

**Pull request merged**